config/main.py

```diff
@@ -342,6 +342,70 @@ def load_minigraph():
     _restart_services()
     print "Please note setting loaded from minigraph will be lost after system reboot. To preserve setting, run `config save`."
 
+
+#
+# 'port_channel' group
+#
+@cli.group()
+@click.pass_context
+def port_channel(ctx):
```

Comment:
In the 'show' command, we use 'portchannel', one word, no underscores, as the subcommand string. We should choose one format and use it everywhere for consistency.

Reply:
i will use portchannel